Anishchenko_V/HW10/Task1.py

```diff
@@ -0,0 +1,19 @@
+class Polygon:
```

Comment:
here is a more optimized version of your code

class Polygon:
    def __init__(self):
        pass


class Rect(Polygon):
    def __init__(self):
        super().__init__()

    @staticmethod
    def square(side1_size, side2_size):
        return side1_size * side2_size